Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.8 
+
+WORKDIR /usr/src/GENIE-Sponsored-Projects
+
+RUN git clone git://github.com/cBioPortal/cbioportal.git ../cbioportal
```

Comment:
If you place this before the COPY, the cbioportal repo will forever stay the same as the one it initially builds.. Unless we turn on "no cache"

Reply:
rearranged to clone repo after COPY